cron/internal/worker/main.go

```diff
@@ -56,25 +56,25 @@ var (
 	// TODO, should probably be its own config/env var, as the checks we want to run
 	// per-platform will differ based on API cost/efficiency/implementation.
 	gitlabDisabledChecks = []string{
-		"Binary-Artifacts",
+		// "Binary-Artifacts",
 		"Branch-Protection",
-		"CII-Best-Practices",
-		"CI-Tests",
-		"Code-Review",
-		"Contributors",
-		"Dangerous-Workflow",
-		"Dependency-Update-Tool",
-		"Fuzzing",
-		"License",
-		"Maintained",
-		"Packaging",
-		"Pinned-Dependencies",
-		"SAST",
+		// "CII-Best-Practices",
+		"CI-Tests", // globally disabled
+		// "Code-Review",
+		"Contributors",           // globally disabled
+		"Dangerous-Workflow",     // not supported on gitlab
+		"Dependency-Update-Tool", // globally disabled, not supported on gitlab
+		// "Fuzzing",
+		// "License",
+		// "Maintained",
+		// "Packaging",
+		// "Pinned-Dependencies",
+		"SAST", // not supported on gitlab
 		// "Security-Policy",
-		"Signed-Releases",
-		"Token-Permissions",
-		"Vulnerabilities",
-		"Webhooks",
+		// "Signed-Releases",
+		"Token-Permissions", /// not supported on gitlab
+		// "Vulnerabilities",
+		"Webhooks", // globally disabled
```

Comment:
Did you check these newly enabled checks with the local cron pipeline?

Reply:
Yep, I use the regular setup but replace the controller init with:

```
go run $(ls cron/internal/controller/*.go | grep -v _test.go) \
    --config cron/internal/emulator/config.yaml \
    cron/internal/data/gitlab-projects-selected.csv
```

The worker seems to process all repos successfully with the new checks enabled.